casanovo/denovo/model_runner.py

```diff
@@ -116,36 +118,52 @@ def train(
             self.loaders.val_dataloader(),
         )
 
-    def evaluate(self, peak_path: Iterable[str]) -> None:
-        """Evaluate peptide sequence preditions from a trained Casanovo model.
+    def log_metrics(self, test_index: AnnotatedSpectrumIndex) -> None:
+        """Log pep_precision and aa_precision
```

Comment:
**nitpick:** Write out the actual words instead of variable names, which are slightly less informative.